web/_js/view.js

```diff
@@ -449,6 +449,21 @@ function initView(){
 					return 0;
 				}
 			break;
+			case "area":
+				sortFunction = function(a, b){
+					// todo: not exactly optimised
+					let areaA = calcPolygonArea(a.path);
+					let areaB = calcPolygonArea(b.path);
+					if (areaA > areaB) {
+						return -1;
+					}
+					if (areaA < areaB) {
+						return 1;
+					}
+						// areaA must be equal to areaB
+					return 0;
+				}
```

Comment:
should be able to `return areaB - areaA`

Reply:
👍thanks